README.md

```diff
@@ -3,31 +3,88 @@
 ## Objectives
 
 1. Learn about requiring external code libraries, called gems, in your Ruby applications. 
-2. Learn how to use Bunlder and a Gemfile to manage gems and their dependencies in your applications. 
+2. Learn how to use Bundler and a Gemfile to manage gems and their dependencies in your applications. 
 
-## What is Bundler?
+
+
+### What are RubyGems?
```

Comment:
Almost! We need to also include what the heck a package is. So something like this:

"Nothing you ever write will be 100% your code. While you probably haven't noticed it every day you use somebody elses code. You didn't write your text editor, you didn't write Ruby, you didn't write your operating system. Those are the types of things that regular users interact with. As a developer though there is a new set of outside code you will work with: Libraries. Libraries (or gems in Ruby parlance) are just bundles of code that someone else wrote for you to integrate into your code base. For example, remember rspec?  That's a gem. Instead of everyone having to re-invent a way to do testing for ruby, initially one person and now hundreds of people have worked together to make a single amazing library that everyone can use. It's open source, and you integrate it using the RubyGems tool. Head over to rubygems.com, there are thousands of gems you can pull from that will make your life easier. That is the power of open source. Together we can create something no single person could make.